tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,57 @@
+import pytest, os
+from datetime import datetime
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+from threading import Thread
+from todo_app.trello_api_calls import create_trello_board, delete_trello_board,\
+    fetch_all_cards
+from todo_app.app import create_app
+
+@pytest.fixture(scope='module')
+def test_app():
+    result = create_trello_board()
```

Comment:
You aren't explicitly loading any environment variables in this file, which suggests that the test expects those to already exist. VSCode's test runner can inject these automatically which might be hiding the problem, but if you run the tests from the command line (with `poetry run pytest`) then the tests will fail

We can fix this by loading `.env` during this setup (before creating the trello board) in a similar fashion to how you loaded `.env.test` in the integration test

Reply:
This caught me unaware - the tests were failing through command line ... Fixed !